doc/res-sendFile.md

```diff
@@ -0,0 +1,80 @@
+<!---
+ Copyright (c) 2016 StrongLoop, IBM, and Express Contributors
+ License: MIT
+-->
+
+<h3 id='res.sendFile'>res.sendFile(path [, options] [, fn])</h3>
+
+<div class="doc-box doc-info" markdown="1">
+`res.sendFile()` is supported by Express v4.8.0 onwards
+</div>
+
+Transfers the file at the given `path`. Sets the `Content-Type` response HTTP header field
+based on the filename's extension. Unless the `root` option is set in
+the options object, `path` must be an absolute path to the file.
+
+The following table provides details on the `options` parameter.
+
+<div class="table-scroller" markdown="1">
+
+| Property        | Description                                     | Default     | Availability |
```

Comment:
It's because the table is inside of 
`<div class="table-scroller" markdown="1"> ...  <div>`

The only way I know how to fix this (and preserve the existing functionality) is to convert the table to HTML.  This would make it a bit harder to edit than a markdown table, but if that's OK, I'll do so....?  

NB: we have several other HTML tables in various places in the doc anyway.